### PR TITLE
add information about HVDC lines

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/components/network/config-tables.js [hunk 1]
```diff
@@ -748,6 +748,37 @@ export const TABLES_DEFINITIONS = {
...
+            },
+            {
+                id: 'K',
+                dataKey: 'k',
+                numeric: true,
+                fractionDigits: 1,
+            },
+            {
+                id: 'AcEmulation',
+                dataKey: 'enabled',
```
Comment (niconoir) [inline src/components/network/config-tables.js:773]: Why not 'acEmulation' as dataKey ? 'enabled' is a bit generic.

File: src/translations/fr.json [hunk 2]
```diff
@@ -209,6 +209,11 @@
     "ConverterStationId2": "ID station conversion 2",
     "NominalV": "Tension nominale (kV)",
     "HvdcLineId": "ID ligne HVDC",
+    "OprFromCS1toCS2": "LIMITE DE FONCTIONNEMENT (COTE_1 VERS COTE_2) (MW)",
+    "OprFromCS2toCS1": "LIMITE DE FONCTIONNEMENT (COTE_2 VERS COTE_1) (MW)",
+    "AcEmulation": "EMULATION AC",
+    "K": "GAIN (MW/deg)",
+    "P0": "P0 (COTE_1 VERS COTE_2) (MW)",
```
Comment (niconoir) [inline src/translations/fr.json:216]: Do not write translations only with uppercases, even if the name of the column in the table is displayed in uppercases. When configuring displayed columns (by clicking on 'Edit columns' button), the names of the columns is not converted to uppercases.

File: src/translations/en.json [hunk 3]
```diff
@@ -209,6 +209,11 @@
     "ConverterStationId2": "Converter Station Id 2",
     "NominalV": "Nominal V",
     "HvdcLineId": "HVDC Line Id",
+    "OprFromCS1toCS2": "Operational limit (side_1 to side_2) (MW)",
+    "OprFromCS2toCS1": "Operational limit (side_2 to side_1) (MW)",
+    "AcEmulation": "Ac emulation",
```
Comment (niconoir) [inline src/translations/en.json:214]: Ac in uppercase (AC)

File: src/translations/fr.json [hunk 4]
```diff
@@ -209,6 +209,11 @@
     "ConverterStationId2": "ID station conversion 2",
     "NominalV": "Tension nominale (kV)",
     "HvdcLineId": "ID ligne HVDC",
+    "OprFromCS1toCS2": "Limite de fonctionnement (cote_1 vers cote_2) (MW)",
+    "OprFromCS2toCS1": "limite de fonctionnement (cote_2 vers cote_1) (MW)",
+    "AcEmulation": "Emulation ac",
```
Comment (niconoir) [inline src/translations/fr.json:214]: Write ac in uppercase (AC)

**Review by niconoir — CHANGES_REQUESTED** [review]
Just a minor remark

**Review by niconoir — APPROVED** [review]

File: src/translations/fr.json [hunk 5]
```diff
@@ -209,6 +209,11 @@
     "ConverterStationId2": "ID station conversion 2",
     "NominalV": "Tension nominale (kV)",
     "HvdcLineId": "ID ligne HVDC",
+    "OprFromCS1toCS2": "Limite de fonctionnement (cote_1 vers cote_2) (MW)",
```
Comment (FranckLecuyer) [inline src/translations/fr.json:212]: côté

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

**Review by FranckLecuyer — APPROVED** [review]

**Pull request merged**